aip/auth/4111.md

```diff
@@ -0,0 +1,84 @@
+--- 
+aip: 
+  id: 4111
+  scope: auth
+  state: approved
+  created: 2018-07-21
+  updated: 2019-07-21
+permalink: /auth/4111
+redirect_from:
+  - /4111
+---
+
+# Self-signed JWT-based Access Token
+
+JWT access token for Google APIs was introduced in 2014 to provide a more
```

Comment:
I recommend we use consistent word choice to describe our concept of "JWT". In this single file, there are 6 variations:
Self-signed JWT-based Access Token
JWT Access Token
JWT-based access token
self-signed JWT
self-signed JWT-based token
JWT

Since JWT stands for JSON Web Tokens, it feels redundant to add another "token" at the end. My recommendation is to use "Self-signed JWT" to describe the concept and the token. Although it is used in a very similar way as OAuth Access Token, it really is not an "Access Token" under the OAuth spec.